aptos-move/writeset-transaction-generator/src/writeset_builder.rs

```diff
@@ -53,13 +54,18 @@ impl<'r, 'l, S: MoveResolver> GenesisSession<'r, 'l, S> {
             });
     }
 
-    pub fn exec_script(&mut self, sender: AccountAddress, script: &Script) {
+    pub fn exec_script(
+        &mut self,
+        sender: AccountAddress,
+        script: &Script,
+    ) -> SerializedReturnValues {
+        let mut temp = vec![sender.to_vec()];
+        temp.extend(convert_txn_args(script.args()));
```

Comment:
This concatenation is so frequently used, shall we consider roll it up into a `SessionExt::execute_script` or a helper function?

Reply:
will pun this for now in future https://github.com/aptos-labs/aptos-core/issues/547